File: dave_package/GPS.py
class GPS:
    NORTH_CENTRAL = "benue, kogi, kwara, nasarawa, niger, plateau, fct"
    NORTH_EAST = "adamawa, bauchi, borno, gombe, taraba, yobe"
    NORTH_WEST = "jigawa, kaduna, kano, katsina, kebbi, sokoto, zamfara"
    SOUTH_EAST = "abia, anambra, ebonyi, enugu, imo"
    SOUTH_SOUTH = "akwa_ibom, bayelsa, cross_river, delta, edo, rivers"
    SOUTH_WEST = "ekiti, lagos, ogun, ondo, osun, oyo"


class Gps:
    @staticmethod
    def get_GPS_state(state):
        lowercase_state = state.lower()
        if lowercase_state in GPS.NORTH_CENTRAL:
            return "NORTH_CENTRAL"
        elif lowercase_state in GPS.NORTH_EAST:
            return "NORTH_EAST"
        elif lowercase_state in GPS.NORTH_WEST:
            return "NORTH_WEST"
        elif lowercase_state in GPS.SOUTH_EAST:
            return "SOUTH_EAST"
        elif lowercase_state in GPS.SOUTH_SOUTH:
            return "SOUTH_SOUTH"
        elif lowercase_state in GPS.SOUTH_WEST:
            return "SOUTH_WEST"
        else:
            return None

    @staticmethod
    def Result():
        state = input("Enter your state: ")
        gps = Gps.get_GPS_state(state)

        if gps:
            print(f"Geopolitical zone: {gps}")
        else:
            print("Invalid State entered!")


Gps.Result()

File: src/Example1/GPS.java
package Example1;

public enum GPS {
    NORTH_CENTRAL("BENUE, KOGI, KWARA, NASARAWA, NIGER, PLATEAU, FCT"),
    NORTH_EAST("ADAMAWA, BAUCHI, BORNO, GOMBE, TARABA, YOBE"),
    NORTH_WEST("JIGAWA, KADUNA, KANO, KATSINA, KEBBI, SOKOTO, ZAMFARA"),
    SOUTH_EAST("ABIA, ANAMBRA, EBONYI, ENUGU, IMO"),
    SOUTH_SOUTH("AKWA_IBOM, BAYELSA, CROSS_RIVER, DELTA, EDO, RIVERS"),
    SOUTH_WEST("EKITI, LAGOS, OGUN, ONDO, OSUN, OYO");

    private String states;

    GPS(String states) {
        this.states = states;
    }

    public String getStates() {
        return states;
    }

    public void setStates(String states) {
        this.states = states;
    }

    public static GPS getGPSState(String state) {
        for (GPS gps : GPS.values()) {
            String[] states = gps.getStates().split(", ");
            for (String s : states) {
                if (s.equalsIgnoreCase(state)) {
                    return gps;
                }
            }
        }
        return null;
    }
}
